corehq/apps/users/dbaccessors/all_commcare_users.py

```diff
@@ -0,0 +1,12 @@
+def get_all_commcare_users_by_domain(domain):
+    """Returns all CommCareUsers by domain regardless of their active status"""
+    from corehq.apps.users.models import CommCareUser
+    key = [domain, CommCareUser.__name__]
+
+    return [CommCareUser.wrap(user['doc']) for user
```

Comment:
this should probably use [iter_docs](https://github.com/dimagi/dimagi-utils/blob/master/dimagi/utils/couch/database.py#L53) - we've observed that performance is significantly better if you do this for anything that can return more than a hundred or so results